src/pages/signup/SignUp.tsx

```
@@ -1,10 +1,76 @@
+      <S.SignUpWrapper>
+        <S.PreviousArea to={`${ROUTES.withEmail}`}>
+          <S.BackToPrevious src={arrow_left} />
+          <S.BackArea>
+            <S.Back>이전</S.Back>
+          </S.BackArea>
+        </S.PreviousArea>
+        <S.ContentArea onSubmit={onSubmit(handleSubmit)}>
+          <S.InputContainer>
+            {SIGN_UP_CONSTANTS.map((constants) => (
+              <S.InputWrapper>
+                <S.Label>{constants.label}</S.Label>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)